tests/core/test_simulation_builder.py

```diff
@@ -349,9 +351,85 @@ def test_unallocated_person(simulation_builder, group_entity):
     assert exception.value.error == {'households': "{'Sarah'} have been declared in persons, but are not members of any household. All persons must be allocated to a household."}
 
 
+def test_nb_persons_in_group_entity(simulation_builder):
+    persons_ids: Iterable = [2, 0, 1, 4, 3]
```

Comment:
@benjello Thanks for asking, we've started using this and they're a great tool which will be even more useful if we take time to communicate and explain :)

These are type hints, a new feature since Python 3, see here: https://docs.python.org/3/library/typing.html

We're using the type hints to document the code internally in places where we would otherwise wonder too often "what Python type is this supposed to be".

Reply:
I do not see this exact syntax in the link you provided. So what are the effect of this line exactly ?
Is it different than 
```python
persons_ids = [2, 0, 1, 4, 3]
```
?